docs/framework/filesystem.md

```diff
@@ -142,40 +142,39 @@ CiviCRM provides two main file storage helpers:
-| `extensionsDir`      | `[civicrm.files]/ext`                | Allow file reads but prohibit directory listing | Stores downloaded extensions. Writes should be infrequent. Reads are very frequent. In hardened systems, this may be readonly or disabled.
-| `imageUploadDir`     | `[civicrm.files]/persist/contribute` | Allow file reads but prohibit directory listing | Stores uploaded or autogenerated media (images/css/etc). Writes and reads vary widely (depending on use-case/config).
-| `templateCompileDir` | `[civicrm.compile]` defaults to `[civicrm.private]/templates_c`| Prohibit all web access                         | Stores autogenerated PHP files. Writes should be infrequent. Reads are very frequent.
-| `uploadDir`          | `[civicrm.files]/upload`             | Prohibit all web access                         | Temporary files. Writes and reads vary widely (depending on use-case/config).
+| Code Name             | Typical Path                                                   | Recommended Access Level                        | Comments                                                                                                                                   |
+| --------------------- | -------------------------------------------------------------- | ----------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------ |
+| `configAndLogDir`     | `[civicrm.log]` defaults to `[civicrm.private]/ConfigAndLog`   | Prohibit all web access                         | Stores log files. Writes and reads should be infrequent, unless there are errors/warnings.                                                 |
+| `customFileUploadDir` | `[civicrm.files]/custom`                                       | Prohibit all web access                         | Stores sensitive uploads (such as custom-field attachments). Writes and reads vary widely (depending on use-case/config).                  |
+| `extensionsDir`       | `[civicrm.files]/ext`                                          | Allow file reads but prohibit directory listing | Stores downloaded extensions. Writes should be infrequent. Reads are very frequent. In hardened systems, this may be readonly or disabled. |
+| `imageUploadDir`      | `[civicrm.files]/persist/contribute`                           | Allow file reads but prohibit directory listing | Stores uploaded or autogenerated media (images/css/etc). Writes and reads vary widely (depending on use-case/config).                      |
+| `l10nDir`             | `[civicrm.l10n]` defaults to `[civicrm.private]/l10n`          | Prohibit all web access                         | Stores CiviCRM's localisation files. Writes should be infrequent (generally on Update or Extension Install). Reads are very frequent.      |
```

Comment:
@MikeyMJCO Just noting I think the l10nDir only relates to the core l10n files not extension ones

Reply:
So the l10n Update extension uses the same core system function to place the files.

https://github.com/cividesk/com.cividesk.l10n.update/blob/8f954e81f76bac0de1066ce3ac74891ea2ea0d1c/l10nupdate.php#L79

Based on my readings so does core?